### ISSUE TITLE
The ampersand character isn't being escaped with XML formatters

### ISSUE THREAD

**Issue by ghost** [CSSLint/csslint#255]
I'm not sure if the Rhino version does this, but the nodejs version doesn't escape the ampersand character when outputting in one of the XML formats.

So for examples like below:

``` css
.someSelector {
  content: "&darr;";
}
```

It outputs `&darr;` directly into the XML file, which causes problems with XML parsers complaining that the `&darr;` entity doesn't exist.  In my particular case, the Violations plugin in Jenkins errors out and fails my build because of this issue.

I've found the section in code which escapes the characters and fixed it, but thought I should report it as an issue first in case there's something I'm missing.  Otherwise I'm happy to submit a pull request.

Chris
